### PR TITLE
VZ-6425: Add HA section under customize installations

### PR DESCRIPTION
**Pull request by mgianatagh**
Add a "High Availability" sub-section to Customize Installations

### CODE REVIEW

File: content/en/docs/setup/customizing/ha.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: High Availability Configuration
+description: Using the `prod` profile to achieve a high availability installation
+linkTitle: High Availability
+Weight: 5
+draft: false
+---
+
+The file [ha.yaml]({{< ghlink raw=true path="examples/ha/ha.yaml" >}}) is an example of how the `prod` profile can be extended to configure a Verrazzano installation to be highly available. The increased replica counts, along with the affinity rules inherited from the `prod` profile, ensure the pods of each component are distributed across the Kubernetes cluster nodes.  There would be no loss of service if a cluster node became unavailable, or if Verrazzano is upgraded.
```
Comment (kminder) [inline content/en/docs/setup/customizing/ha.md:9]: Is the "or if Verrazzano is upgrade" part really true.  I don't think we really test that right?

File: content/en/docs/setup/customizing/ha.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: High Availability Configuration
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ha.md]: title: Configure High Availability

File: content/en/docs/setup/customizing/ha.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+description: Using the `prod` profile to achieve a high availability installation
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ha.md:3]: description: Achieve a highly available installation using the `prod` profile. 
OR (can't decide which I like better; you can decide) 
description: Achieve high availability using the `prod` profile.

File: content/en/docs/setup/customizing/ha.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+linkTitle: High Availability
+Weight: 5
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ha.md:5]: weight: 11

File: content/en/docs/setup/customizing/ha.md (hunk 1 above)
Comment (mgianatagh) [reply content/en/docs/setup/customizing/ha.md:9]: We are not testing upgrade yet.  I will remove the upgrade portion.

Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ha.md]: Suggested re-write of the first sentence, "The [ha.yaml]({{< ghlink raw=true path="examples/ha/ha.yaml" >}}) file shows you how the `prod` profile can be extended to configure a highly available Verrazzano installation.

File: content/en/docs/setup/customizing/ha.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+title: Configure High Availability
+description: Using the `prod` profile to achieve a high availability installation
+linkTitle: Configure High Availability
+Weight: 5
+draft: false
+---
+
+The file [ha.yaml]({{< ghlink raw=true path="examples/ha/ha.yaml" >}}) is an example of how the `prod` profile can be extended to configure a Verrazzano installation to be highly available. The increased replica counts, along with the affinity rules inherited from the `prod` profile, ensure the pods of each component are distributed across the Kubernetes cluster nodes.  There would be no loss of service if a cluster node became unavailable.
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ha.md:9]: Suggested re-write of the first sentence, "The [ha.yaml]({{< ghlink raw=true path="examples/ha/ha.yaml" >}}) file shows you how the `prod` profile can be extended to configure a highly available Verrazzano installation.
ensure the pods -> ensure that the pods

File: content/en/docs/setup/customizing/ha.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+When using [ha.yaml]({{< ghlink raw=true path="examples/ha/ha.yaml" >}}), consider the following:
+
+* It does not ensure a fault-tolerant environment
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ha.md:13]: Please put a period at the end of each bulleted item.

File: content/en/docs/setup/customizing/ha.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+* Additional customizations may be required for your environment
+* Running additional replicas of components will increase resource requirements
+
+How to install the example high availability configuration using the Verrazzano CLI:
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ha.md:17]: How to install -> To install

**Review by rosemarymarano — COMMENTED** [review]
Some edits and suggestions; you may have gotten some of them already.

File: content/en/docs/setup/customizing/ha.md (hunk 3 above)
Comment (mgianatagh) [reply content/en/docs/setup/customizing/ha.md:3]: I will go with your description

**Review by rosemarymarano — APPROVED** [review]
Perfect, thank you.

**Pull request merged**